Controllers/controllers.js

```diff
@@ -33,3 +34,12 @@ exports.getArticleById = (req, res, next) => {
       next(err);
     });
 };
+exports.getCommentsByArticle = (req, res, next) => {
+  const {article_id} = req.params
+  selectCommentsByArticle(article_id)
+  .then((comment) =>{
+    res.status(200).send({comment})
+  }).catch((err) => {
+    next(err)
+  })
+}
```

Comment:
As Rob mentioned it in the previous pr, you need to check if the article exists so we get a 404 instead of a 200 and an empty array:

robert112233 1 hour ago

'Here you will need to use a checkIfArticleExists function to differentiate from an invalid article and a valid article with no comments. See notes for help:

https://notes.northcoders.com/courses/js-back-end/advanced-error-handling'